### PR TITLE
Fix local sourcemaps

### LINKED ISSUE #1174: Can't view source map using `npm run watch`
Example of trying to set breakpoint when inspecting the page editor:

image

> Could not load content for webpack:///./src/devTools/editor/extensionPoints/trigger.tsx (HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME)

@fregante any ideas?

### PR DESCRIPTION
**Pull request by fregante**
- Fixes #1174 

Screen Shot 1

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -308,9 +308,6 @@ module.exports = (env, options) =>
         new webpack.SourceMapDevToolPlugin({
           publicPath: sourceMapPublicUrl,
 
-          // Produce full/heavy sourcemaps only if they're going to be public
```
Comment (twschiller) [inline webpack.config.js:311]: Will this now create the source maps on branch/PR CI runs? Do we need to check for the CI env variable here?

**Review by twschiller — CHANGES_REQUESTED** [review]
Confirm behavior on CI (see other comment)

File: webpack.config.js (hunk 1 above)
Comment (fregante) [reply webpack.config.js:311]: No this only affects the _quality_ of the sourcemaps.

- CI: no source maps
- CI on `main`: full external sourcemaps on S3
- local builds: full inline sourcemaps

Testing already suggests using `CI=true` to get no sourcemaps to get a lighter zipped extension for BrowserStack.

**Review by twschiller — APPROVED** [review]

**Comment by twschiller** [comment]
image

**Pull request merged**